trunner/test_runner.py

```diff
@@ -158,7 +158,10 @@ def flash(self) -> TestResult:
             print(exc)
             result.fail_harness_exception(exc)
 
-        print("Done!")
+        self._print_test_header_end(result)
+
+        print(result.status)
```

Comment:
I think that we should use `result.to_str` as it's done in tests, because now it's not consistent, we want to also have `flash: FAIL` and then output in such case:

![Screenshot from 2024-04-16 12-35-48](https://github.com/phoenix-rtos/phoenix-rtos-tests/assets/77304431/45b4a714-4f4c-4011-a948-962e9bd5913a)


![Screenshot from 2024-04-16 12-36-00](https://github.com/phoenix-rtos/phoenix-rtos-tests/assets/77304431/1d5f8964-efea-4928-91a3-d75608541a9b)

Reply:
After the recent changes, it now looks like this:
![image](https://github.com/phoenix-rtos/phoenix-rtos-tests/assets/116297248/52eece1a-8441-4a89-9a64-9301eb6e8aeb)